app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -168,6 +168,15 @@ def remove
       respond_for_async :resource => task
     end
 
+    api :PUT, "/content_views/:id/remove_filters", N_("Delete multiple filters from a content view")
+    param :id, :number, :desc => N_("content view numeric identifier"), :required => true
+    param :filter_ids, Array, of: :number, :desc => N_("filter identifiers"), :required => true
+    def remove_filters
+      @content_view = Katello::ContentView.readable.find_by(:id => params[:id])
```

Comment:
Probably don't need this..The `find_authorized_katello_resource` method on top should get you this.

Reply:
For context this method was added during permissions refactoring that happened a few months ago. This finds the controller resource with permissions as long as the call has a params[:id] and other fun things like controller name being in a particular format. Earlier we would've a private method like find_content_view to do the same making it more obvious.